db_setup/schema.sql

```diff
@@ -0,0 +1,109 @@
+USE plants;
+GO
+
+DROP TABLE if exists s_beta.watering, s_beta.duty, s_beta.recording, s_beta.botanist, s_beta.plant, s_beta.location;
+GO
+
+CREATE TABLE s_beta.recording (
+    "id" INT IDENTITY(1,1),
+    "plant_id" INT NOT NULL,
+    "soil_moisture" FLOAT NOT NULL,
+    "temperature" FLOAT NOT NULL,
+    "datetime" datetimeoffset NOT NULL
+);
+GO
+
+ALTER TABLE
+ s_beta.recording ADD CONSTRAINT "recording_id_primary" PRIMARY KEY("id");
+GO
+
+CREATE TABLE s_beta.plant(
+    "id" INT IDENTITY(0,1),
+    "name" VARCHAR(255) NOT NULL,
+    "scientific_name" VARCHAR(255) NOT NULL,
+    "location_id" INT
+);
+GO
+
+ALTER TABLE
+    s_beta.plant ADD CONSTRAINT "plant_id_primary" PRIMARY KEY("id");
+GO
+
+CREATE TABLE s_beta.watering(
+    "id" INT IDENTITY(1,1),
+    "plant_id" INT NOT NULL,
+    "datetime" datetimeoffset NOT NULL
+);
+GO
+
+ALTER TABLE
+    s_beta.watering ADD CONSTRAINT "watering_id_primary" PRIMARY KEY("id");
+GO
+
+
+CREATE TABLE s_beta.botanist(
+    "id" INT IDENTITY(1,1),
+    "email" VARCHAR(255) NOT NULL,
+    "firstname" VARCHAR(255) NOT NULL,
+    "lastname" VARCHAR(255) NOT NULL,
+    "phone" VARCHAR(255) NOT NULL
```

Comment:
really not a big deal, but could change phone to be VARCHAR(20) and emails can be longer so maybe VARCHAR(350)

Reply:
Will do!